client/components/domains/connect-domain-step/connect-domain-step-suggested-start.jsx

```diff
@@ -48,6 +47,35 @@ export default function ConnectDomainStepSuggestedStart( {
 				<br />
 				{ __( 'It can take up to 72 hours for the domain to be fully connected.' ) }
 			</p>
+			<FoldableCard
```

Comment:
Minor: I noticed that SVG icon is slightly different in Figma, you can copy from `client/components/domains/accordion/index.tsx`

```
actionButton={
	<button className="foldable-card__action foldable-card__expand">
		<span className="screen-reader-text">More</span>
		<Icon icon={ chevronDown } viewBox="6 4 12 14" size={ 16 } />
	</button>
}
actionButtonExpanded={
	<button className="foldable-card__action foldable-card__expand">
		<span className="screen-reader-text">More</span>
		<Icon icon={ chevronUp } viewBox="6 4 12 14" size={ 16 } />
	</button>
}
```

Reply:
Thanks, good catch! I'll update this